js/main.js

```diff
@@ -11,12 +11,13 @@ APP.init = function() {
 	);
 }
 
-APP.getSessions = function(client, token, email) {
+APP.getSessions = function(client, email) {
 	var sessions = {
-		url:'https://www.smartlook.com/api/sessions.list',
+		url: 'https://www.smartlook.com/api/sessions.list',
```

Comment:
Try it please with `x-apikey` header and url https://api-gateway.smartlook.com/legacy/api/sessions.list. We would like to get rid of www.smartlook.com/api.

Reply:
The header has been set, the endpoint was finally set to https://fiona.smartlook.com/api/sessions.list